README.md

```diff
@@ -1,5 +1,99 @@
-# q2-chemistree
-
-[![Build Status](https://travis-ci.org/biocore/q2-chemistree.svg?branch=master)](https://travis-ci.org/biocore/q2-chemistree) [![Coverage Status](https://coveralls.io/repos/github/biocore/q2-chemistree/badge.svg?branch=master)](https://coveralls.io/github/biocore/q2-chemistree?branch=master)
-
-Hierarchical orderings for mass spectrometry data
+# q2-chemistree
+
+A tool to build a tree of MS1 features for use with phylogenetic methods in metabolomics datasets.
+
+## Installation
+
+Once QIIME2 is [installed](https://docs.qiime2.org/2018.2/install/), activate your QIIME2 environment and install q2-chemistree following the steps below:
+
+```bash
+git clone https://github.com/biocore/q2-chemistree.git
+cd q2-chemistree
+pip install .
+qiime dev refresh-cache
+```
+
+We also need to download SIRIUS, a software-framework for de-novo identification of metabolites. q2-chemistree uses molecular substrucures predicted by this framework to build a hierarchy of the MS1 features in a dataset. SIRIUS is freely available [here](https://bio.informatik.uni-jena.de/software/sirius/).

```

Comment:
In the first mention of SIRIUS can you make that word a hyperlink to their homepage?